src/commands/ticker/index.ts

```diff
@@ -36,6 +36,7 @@ const textCmd: Command = {
   allowDM: true,
   run: async function (msg) {
     const args = getCommandArguments(msg)
+    const chain = args[2] ?? ""
```

Comment:
k cần fallback sớm ở đây nếu e không còn làm gì khác với variable này ngoài việc pass nó vào function kế tiếp
```suggestion
    const chain = args[2]
```